fedimint-core/src/tiered_multi.rs

```diff
@@ -62,20 +62,20 @@ impl<T> TieredMulti<T> {
         self.count_items() == 0
     }
 
-    /// Applies the given closure to every `(Amount, T)` pair
-    pub fn map<F, N, E>(self, f: F) -> Result<TieredMulti<N>, E>
```

Comment:
Thought: Why do we even need it, if we have `iter()` and `IntoIterator`?

Reply:
Good point! Done, we're now calling `TieredMulti::iter_items()` and I removed `TieredMulti::map()`.